dnf/automatic/main.py

```diff
@@ -217,6 +217,17 @@ def __init__(self):
         self.add_option('system_name', libdnf.conf.OptionString(socket.gethostname()))
 
 
+def gpgsigcheck(base, pkgs):
+    ok = True
+    for po in pkgs:
+        result, errmsg = base._sig_check_pkg(po)
```

Comment:
I am sorry, but I would prefer to create a new API for DNF that will check or import gpg keys. There are more API users that requires the same functionality (DNF dragora).

Reply:
Added commit with new public API for GPG handling.